external/Get-GitModule.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+  .SYNOPSIS
+  Downloads the Git modules specified in ../.gitmodules
+
+  .DESCRIPTION
+  Parses and downloads the Github repositories specified in the .gitmodules file
+
+  .EXAMPLE
+  PS> .\Get-GitModules.ps1
```

Comment:
Little typo in the file name. It should be plural, so "Get-GitModule**s**.ps1".

Reply:
Good catch, thank you, it should be Get-GitModules.ps1 now.